package.json

```diff
@@ -43,7 +43,7 @@
     "redent": "^2.0.0"
   },
   "devDependencies": {
-    "jsdom": "^12.2.0",
```

Comment:
@gnapse In fact, I am just realizing that the jsdom library was already part of this project.
I am not sure I would need this update. Will rollback to 12.2 and seeing if it's still working.

If it's the case, I should be able to remove the "Warning" comment in the test file.